app/ui.js

```diff
@@ -1745,10 +1745,9 @@ const UI = {
                     if (UI.rfb) {
                         UI.rfb.forcedResolutionX = event.data.value_x;
                         UI.rfb.forcedResolutionY = event.data.value_y;
+                        UI.forceSetting('forced_resolution_x', event.data.value_x, false);
```

Comment:
When selecting Scaling Mode Off and setting the resolution it's bringing up an error

<img width="1693" alt="image" src="https://github.com/kasmtech/noVNC/assets/5698566/ecc5faa6-5b9e-4746-ac80-70b27de893f8">

Reply:
This might be because I'm running it on 192.168.64.3 rather than a proper domain as it looks like these are looking for cookie settings?

Reply:
https://www.loom.com/share/67d715b432db4ef6bf8f137764115028?sid=230e39ed-0849-4b33-925a-f66e28c6bcf0

Reply:
@KodeStar , I believe this has been resolved, please retest. Due to the changes, you should retest a lot of different configuration.